documentation/modules/configuring/con-skipping-scale-down-checks.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// con-scaling-kafka-clusters.adoc
+
+[id='con-skipping-scale-down-checks-{context}']
+= Skipping checks on scale-down operations
+
+[role="_abstract"]
+By default, Strimzi performs a check to ensure that there are no partition replicas on a broker before initiating a scale-down operation on a Kafka cluster. 
+If replicas are found, the scale-down operation is blocked to prevent potential data loss. 
+To fix this, no replicas must be left on the broker before trying to scale it down again.
+
+However, there may be scenarios where you want to bypass this blocking mechanism.
+Disabling the check might be necessary on busy clusters, for example, because new topics keep generating replicas for the broker.
```

Comment:
This is fine. But you should also add that when the broker is scaled down with some topics still on it, it will likely caused the reconciliation fail a little bit later. Talking about only one side of the issue makes it sound like a cleat choice which it is not.

Reply:
Added:  _Overriding the blocking mechanism in this way has an impact: the presence of topics on the broker being scaled down will likely cause a reconciliation failure for the Kafka cluster._